### ISSUE TITLE
Volume control

### ISSUE THREAD

**Issue by yellowbird911** [FrSky-OS/discussion#70]
Would be great if in the sound menu it would allow you to assign a knob to control telemetry volume.  May be able to set this up in Special Functions but from a new user stand point it would be nice.

**Comment by AloftHobbies** [comment]
Agreed, that would be a much nicer method to do that. I always have a pot used for volume control on my Taranis. As another option, just touch a speaker icon on the Horus touch screen?
